### PR TITLE
Remove the nullState

### PR DESCRIPTION
**Pull request by duglin**
Add a "createdState" in its place since I think that better describes
what its used for.

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by mikebrow** [comment]
Reviewed code... no issues found.

File: libcontainer/container.go [hunk 1]
```diff
@@ -43,7 +49,7 @@ func (s Status) String() string {
 	case Destroyed:
 		return "destroyed"
 	default:
-		return "undefined"
+		panic(fmt.Sprintf("Should never get here: %#v", s))
```
Comment (crosbymichael) [inline libcontainer/container.go]: I would rather not have a panic if we can prevent it

Comment (duglin) [inline libcontainer/container.go]: I was thinking about removing the entire "default" case but wasn't sure if we needed to worry about some other random value showing up.  If you're ok with me removing it then I will.

Comment (crosbymichael) [inline libcontainer/container.go]: you have to have a default return value somehow we just should not panic

Comment (duglin) [inline libcontainer/container.go]: ok fixed

**Comment by crosbymichael** [comment]
LGTM

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**